src/config/__snapshots__/typeConfigs.test.ts.snap

```diff
@@ -274,19 +274,13 @@ Array [
     ],
     "component": [Function],
     "description": "Open a ticket with a human agent",
-    "filter": "ticketer",
     "form": [Function],
     "localization": [Function],
     "localizeableKeys": Array [
       "exits",
     ],
     "name": "Open Ticket",
-    "type": "open_ticket",
-    "visibility": Array [
-      "messaging",
-      "messaging_background",
-      "voice",
-    ],
+    "type": "open_wenichats_ticket",
```

Comment:
Should the current Open Ticket test be modified? Or a new test for open_wenichats_ticket be created leaving this one as it as?